documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -297,290 +344,610 @@ the position of the plain you want to cut. These fields are ``u`` and ``v``:
+In order to construct a spherical projection (i.e. a projection in |Q|,
+:math:`\theta` (polar angle), :math:`\phi` (azimuthal angle), :math:`E`) we
+create a projection in an analagous way to the ``line_proj``, but using the
+``sphere_proj`` function:
+
+The complete signature for ``sphere_proj`` is:
 
 .. code-block:: matlab
- 
-    sp_pr = sphere_proj()
-    sp_pr =      
-       sphere_proj with properties:   
-             ez: [1 0 0]
-             ex: [0 1 0]
-           type: 'add'
-          alatt: []
-         angdeg: []
-         offset: [0 0 0 0]
-          label: {'|Q|'  '\theta'  '\phi'  'En'}
-          title: ''
-
-Projection's lattice parameters (``alatt`` and ``angdeg``) are taken within the ``cut`` from source ``sqw`` object. 
-You need to set them up manually only if you intend to use the projection separately in your custom code.
-
-The projection defines spherical coordinates system, where :math:`\theta` angle is 
-measured from z-axis directed along :math:`e_z` vector of the 
-projection and changes from :math:`0` to :math:`180^o`. :math:`\phi` angle is measured
-from :math:`r_x` vector of the projection and changes from :math:`-180^o` to :math:`180^o`. 
-Spherical projection :math:`r_x` vector corresponds to ``sphere_proj`` class :math:`e_x` vector
-for orthogonal reciprocal lattice. If reciprocal lattice is non-orthogonal, the :math:`e_x`-vector
-of spherical projection defines the :math:`z-x` plane of spherical coordinate system and :math:`r_x` 
-vector is build in this plane but orthogonal to :math:`e_z` vector.
-
-.. figure:: ../images/spher_coordinates.png 
+
+   proj = sphere_proj([ez][, ex][, type][, alatt][, angdeg][, offset][, label][, title][, lab1][, lab2][, lab3][, lab4])
+
+where:
+
+- ``ez``
+
+  is :math:`e_z` the 3-vector defining the direction of the :math:`z`-axis in
+  reciprocal space of the spherical coordinate system.
+
+- ``ex``
+
+  is :math:`e_x` the 3-vector defining the direction of the :math:`x`-axis in
+  reciprocal space of the spherical coordinate system.
+
+.. note::
+
+   The :math:`e_y` direction is not explicitly defined as part of the
+   ``sphere_proj`` and is always considered to be perpendicular to the
+   :math:`e_x`-:math:`e_z` plane.
+
+.. note::
+
+   By default a ``sphere_proj`` will define its principal axes
+   :math:`e_z` and :math:`e_x` along the :math:`hkl` directions
+   :math:`[1,0,0]` (:math:`e_z`) and :math:`[0,1,0]` (:math:`e_x`)
+   respectively.
+
+- ``type``
+
+
+  Three character string denoting the the projection normalization of each
+  dimension, one character for each, e.g. ``'add'``, ``'rrr'``, ``'pdr'``.
+
+  There are 3 possible options for the first (length) component of ``type``,
+  which are the same as those for ``line_proj``:
+
+  1. ``'a'``
+
+     Inverse angstroms
+
+  2. ``'r'``
+
+     Reciprocal lattice units (r.l.u.) which normalises so that the maximum of
+     :math:`|h|`, :math:`|k|` and :math:`|l|` is unity.
+
+  3. ``'p'``
+
```

Comment:
Everything excect `a`  is not currently there and I am not sure what is written here now has physical meaning.  certainly `p` would mention `e_z` if implemented as `u` and `v` are not used in `sphere_proj` and `r` will also mean other thing. Instead of guessing now, I would just remove it.

Reply:
May be `r` would mean normalizing `e_z` by its length in Angstroms and `p` is irrelevant as `e_z` is always directed along specified direction, so currently I see only two reasonable meaning for first value